README.md

```diff
@@ -330,6 +330,14 @@ can give prompt as:
 f.input :age, collection: 18..60, prompt: "Select your age"
 ```
 
+Or you can set the selected option as well:
+
+```ruby
+f.input :age, collection: 18..60, selected: 21
+```
+The input value in option `select` will be passed into rails helper without modification.
```

Comment:
I think you meant `selected`? We don't need to make this paragraph specific to selected, as we can simply say that all these extra options are passed to collection select.